README.md

```diff
@@ -1 +1,16 @@
-# tree-sitter-crystal
+# Crystal Treesitter
```

Comment:
```suggestion
# tree-sitter-crystal

```

I'm not sure this should be changed as this is how the other tree-sitter implementations have the header in the README